harness/context_manager.py

```diff
@@ -634,8 +638,9 @@ def apply_patch(
         logger_taskenv.info(
             f"[{self.testbed_name}] [{self.instance[KEY_INSTANCE_ID]}] {log_cmd} patch successful ({patch_type})"
         )
-        with open(self.log_file, "a") as f:
-            f.write(f"{APPLY_PATCH_PASS} ({patch_type})\n")
+        if not log_apply_patch_success: # don't log this if we're reverting the patch
```

Comment:
we do 4 calls,

1. apply model patch
2. revert model patch (this was just to check whether they successfully applied)
3. apply test patch
4. apply model patch (to actually run eval)

1. and 2. are incorrectly logging when the patch is applied successfully, which breaks the monitor script.

Reply:
When we do the four calls in [`evaluate_predictions`](https://github.com/princeton-nlp/SWE-bench/blob/main/swebench/harness/engine_evaluation.py#L79), the the patch_type varies for the four:
* Call 1 - `PATCH_PRED_TRY`
* Call 2 - Revert `PATCH_PRED_TRY`
* Call 3 - `PATCH_TEST`
* Call 4 - `PATCH_PRED`

The [log parsing function](https://github.com/princeton-nlp/SWE-bench/blob/main/swebench/metrics/getters.py#L32) that looks for the test results should then split by the Call 3 and Call 4 log values (e.g. `Applied Patch (test)` and `Applied Patch (pred)`.